EndToEndExample/actions/predict/requirements.txt

```diff
@@ -0,0 +1,7 @@
+pandas
+numpy
+cortex-python==6.0.2a3
```

Comment:
Same here

Reply:
Better to add the versions for all the added requirements to make sure that the code doesn't break with other versions.

Reply:
done